docs/zh/SystemDesign/DataQuery/SeriesReader.md

```diff
@@ -163,55 +163,55 @@ Object[] values = readerByTimestamp.getValueInTimestamp(timestamps, length);
 ```
 
 /*
- * 文件层
- */
-private final List<TsFileResource> seqFileResource;
-	顺序文件列表，因为顺序文件本身就保证有序，且时间戳互不重叠，只需使用 List 进行存储
-	
-private final PriorityQueue<TsFileResource> unseqFileResource;
-	乱序文件列表，因为乱序文件互相之间不保证顺序性，且可能有重叠，为了保证顺序，使用优先队列进行存储
+* 文件索引
+*/
```

Comment:
this change to be QueryDataResource, not the fileIndex.  FileIndex is the index.